classes/abstracts/ActionScheduler_Store.php

```diff
@@ -208,6 +209,40 @@ abstract public function get_claim_id( $action_id );
 	 */
 	abstract public function find_actions_by_claim_id( $claim_id );
 
+	/**
+	 * Check if group name is prefixed for exclussion.
+	 *
+	 * @param string $slug Group slug name.
+	 * @return boolean
+	 */
+	protected static function group_has_ignore_prefix( $slug ) {
+		return self::IGNORE_GROUP_FLAG === substr( $slug, 0, strlen( self::IGNORE_GROUP_FLAG ) );
+	}
+
+	/**
+	 * Mark group for exclussion by prefixing with '-'.
+	 *
+	 * @param string $slug Group slug name.
+	 * @return string
+	 */
+	public static function mark_group_for_exclussion( $slug ) {
```

Comment:
We should change to:

```suggestion
	public static function mark_group_for_exclusion( $slug ) {
```

(And update call-site.)